scripts/managed-services/get-kafka-instance-state/metadata.yaml

```diff
@@ -0,0 +1,30 @@
+file: get-kafka-instance-state.sh
+name: managed-services/get-kafka-instance-state
+description: Print out the logs for the specified Kafka instance. `oc logs <pod-running-job> | tar -xzf -` will create a directory with all instance information.
+author: racheljpg
+allowedGroups:
+  - SREP
+  - CSSRE
+  - CEE
+rbac:
+  roles:
+    - namespace: "redhat-kas-fleetshard-operator"
+      rules:
+        - verbs:
+            - "get"
+            - "list"
+          apiGroups:
+            - ""
+          resources:
+            - "pods/exec"
+        - verbs:
+            - "create"
+          apiGroups:
+            - ""
+          resources:
+            - "pods/exec"
+envs:
+  - key: "kafkaId"
```

Comment:
Maybe add something to this script that will show what kafka IDs exist in this cluster (this making the kafkaid optional)

Reply:
CEE will know what you are calling `kafkaId` as `resource_namespace` (from the MK ID Finder).  You should use the same terminology when naming the argument.